src/backends/tensorrt/plan_backend.cc

```diff
@@ -3044,6 +3041,21 @@ PlanBackend::Context::Run(
     nvinfer1::Dims dims;
     dims = citr->second.context_->getBindingDimensions(binding_index);
 
+    // Wait for model execution to process the output tensors with pinned memory
```

Comment:
The comment here needs to be updated..  No need to discuss waiting here. It is already documented at the ProcessTensor below.

Reply:
^^^ And the waiting is no longer the case: memcpy is async and sync in collector in zero-copy case, the same as the handling in the regular case